ansible/roles/bastion/tasks/k8s.yml

```diff
@@ -0,0 +1,18 @@
+---
+- tags:
+    - bastion_k8s
+  block:
```

Comment:
@jimrigsbee to which PR this file should go? We could not identify the PR related to that change